### PR TITLE
use latest, change default SIF_NAME, change ln command

### PR DESCRIPTION
**Pull request by Thomas-Ulrich**
Hi,
- I don't see the point of not using latest for the non-arm. Therefore changed.
- removed job.jupyter and document the workflow described on the training slides, based on a VNC job.
- fix symbolic link command `ln -sf $(realpath my-training.sif) ~/my-training.sif`
- rename source_modules.sh → setup_modules_Frontera_vnc.sh (more explicit name)

### CODE REVIEW

**Comment by Aangniu** [comment]
Will this directory `$SHARED_PATH/$SIF_NAME` work here?
```
#
SHARED_PATH="/your/path/to/container/"
SIF_NAME="$HOME/my-training.sif"

if [ ! -f $SIF_NAME ]; then
    if [ ! -f $SHARED_PATH/$SIF_NAME ]; then
```

**Comment by Thomas-Ulrich** [comment]
> Will this directory `$SHARED_PATH/$SIF_NAME` work here?
> 
> ```
> #
> SHARED_PATH="/your/path/to/container/"
> SIF_NAME="$HOME/my-training.sif"
> 
> if [ ! -f $SIF_NAME ]; then
>     if [ ! -f $SHARED_PATH/$SIF_NAME ]; then
> ```

@Aangniu you are right. See proposed change.

File: frontera.md [hunk 1]
```diff
@@ -12,18 +12,19 @@ Then execute:
 
 ```
 module load tacc-apptainer
-apptainer pull -F docker://seissol/training:hps-2024-frontera
+apptainer pull -F docker://seissol/training:latest
 apptainer build -f my-training.sif singularity.def
```
Comment (sebwolf-de) [inline frontera.md:16]: Pull and build seem to be duplicate here.

Comment (Thomas-Ulrich) [reply frontera.md:16]: ok so we should remove the line `apptainer pull -F docker://seissol/training:latest` ?

**Review by sebwolf-de — APPROVED** [review]

**Pull request merged**